opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/ErrorMapper.scala

```diff
@@ -2,11 +2,11 @@ package zio.telemetry.opentelemetry.tracing
 
 import io.opentelemetry.api.trace.StatusCode
 
-case class ErrorMapper[E](body: PartialFunction[E, StatusCode])
+case class ErrorMapper[E](body: PartialFunction[E, StatusCode], toThrowable: Option[E => Throwable])
 
 object ErrorMapper {
 
   def default[E]: ErrorMapper[E] =
-    ErrorMapper[E](Map.empty)
+    ErrorMapper[E](Map.empty, None)
```

Comment:
So here you don't need to provide `None` explicitily.

Reply:
fixed